src/shell/shell_cmds.cpp

```diff
@@ -2532,6 +2492,93 @@ static std::vector<std::string> search_files(const std::string_view query)
 	return files;
 }
 
+static DOS_Filename split_extension(const std::string& fullname)
+{
+	DOS_Filename ret = {};
+	size_t pos       = fullname.rfind('.');
+	ret.name         = fullname.substr(0, pos);
+	++pos;
+	if (pos > 0 && pos < fullname.size()) {
+		ret.ext = fullname.substr(pos);
+	} else {
+		ret.ext = {};
+	}
+	return ret;
+}
+
+static std::string handle_wildcards(const std::string& wildcards,
+                                    const std::string& old_filename)
+{
+	std::string ret = {};
```

Comment:
Maybe call it what it is? `expanded_name`?

Reply:
Renamed

Reply:
Or wait... I renamed the wrong one... 1 sec lol

Reply:
OK, now it's renamed :sweat_smile: